_episodes_rmd/07-control-flow.Rmd

```diff
@@ -430,14 +430,14 @@ output_vector2
 > > ```
 > >
 > > We will now store these countries in a variable called candidateCountries, and then loop over each entry in the variable.
-> > Inside the loop, we evaluate the average life expectancy for each country, and if the average life expectancy is less than 50 we use base-plot to plot the evolution of average life expectancy:
+> > Inside the loop, we evaluate the average life expectancy for each country, and if the average life expectancy is less than 50 we use base-plot to plot the evolution of average life expectancy using `with` and `subset`:
```

Comment:
When describing functions we are trying to consistently include the parentheses -- so `with()` and `subset()` instead of the bare function names.

Reply:
Working on this today. FYI, There are a number of places in episode 7 where an R function is given and not followed by parentheses (e.g. lines 354 & 355- use of `if`, lines 418, 426,  use of `grep`. On line 339, `subset` is enclosed in backticks without parentheses, although in that case, it refers to the general concept of a data subset, not the actual R function. I fixed these when the term was clearly referring to a function.